src/components/PopularRecipes/PopularRecipes.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from "react";
+import RecipeCard from "../RecipeCard/RecipeCard";
+import * as api from "../../utils/api";
+import "./PopularRecipes.css";
+
+function PopularRecipes({
+  handleRecipeSummaryOpen,
+  handleSaveRecipe,
+  recommended,
+  setRecommended,
+}) {
+  useEffect(() => {
+    const setPopularRecipes = async () => {
+      try {
+        const results = await api.getPopularRecipes();
+        if (results) {
+          setRecommended(results.recipes);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    setPopularRecipes();
+  }, []);
+
+  return (
+    <section className="popularRecipes">
+      <h3 className="popularRecipes__subheader">
+        Check out these popular Recipes...
+      </h3>
+      <div className="popularRecipes-results__wrapper">
+        <ul className="popularRecipes__cards__list">
+          {recommended?.map((recipe) => (
```

Comment:
Please, use correct BEM naming

There are some `BEM` mistakes `. 


 More info about `BEM` naming can be found here: https://en.bem.info/methodology/quick-start/